docs/data-sources/template.md

```diff
@@ -1,52 +1,252 @@
 ---
-page_title: "template Data Source - terraform-provider-anxcloud"
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "anxcloud_template Data Source - terraform-provider-anxcloud"
 subcategory: ""
 description: |-
-  The template data source allows you to retrieve information about available templates for specified location.
+  Provides available templates for specified location. This information can be used to provision virtual servers using the anxcloud_virtual_server resource.
 ---
 
-# Data Source `anxcloud_template`
+# anxcloud_template (Data Source)
 
-The template data source allows you to retrieve information about available templates for specified location.
+Provides available templates for specified location. This information can be used to provision virtual servers using the `anxcloud_virtual_server` resource.
 
 ## Example Usage
 
-```hcl
+```terraform
+data "anxcloud_core_location" "anx04" {
+  code = "ANX04"
+}
+
 data "anxcloud_template" "example" {
-  location_id = "52b5f6b2fd3a4a7eaaedf1a7c019e9ea"
+  location_id = data.anxcloud_core_location.anx04.id
 }
 ```
```

Comment:
let's add an example how to get a `Debian 11` template identifier, maybe even the latest `Debian 11` template?